pkgs/applications/misc/sandbar/default.nix

```diff
@@ -0,0 +1,44 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  pixman,
+  fcft,
+  pkg-config,
+  wayland-protocols,
+  wayland-scanner,
+  wayland
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "sandbar";
+  version = "v0.1";
+
+  src = fetchFromGitHub {
+    owner = "kolunmi";
+    repo = "sandbar";
+    rev = finalAttrs.version;
+    hash = "sha256-uG+/e75s/OQtEotR+8aXTEjW6p3oJM8btuRNgUVmIiQ=";
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+    wayland-protocols
+    wayland-scanner
+    fcft
+    pixman
+    wayland
```

Comment:
Please test again to see which packages are needed in `nativeBuildInputs`. Seems some are not needed. Also you can enable `strictDeps = true` (I suspect `wayland-protocols` and `wayland-scanner` should also go to `buildInputs`)